charts/generic-data-analytics-extractor/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Declare variables to be passed into your templates.
+
+image:
+  repository: ministryofjustice/data-engineering-data-extractor
+  tag: v1
+
+# Run once daily at 1am
+cronJobSchedule: "0 1 * * *"
+awsDefaultRegion: eu-west-2
+saveEventsLog: "true"
```

Comment:
do we know what this `saveEventsLog ` is used for?

Reply:
We think it is used to save the event logs to the s3 bucket and is referenced by the python scripts in the data-engineering-data-extractor repo. We could set it to false as a default and override it where needed?